.eslintignore

```diff
@@ -1,2 +1,2 @@
-node_modules
-transformations
```

Comment:
@sepehrfakour Why not using an editor that will automatically reformat your code for you so that you don't have to think about this? Then add to gitignore: tranformations/*.local.js so that you can have local transformations without having to have a unusual git config (folder is in git but can't add any other file to it)?

Here's how it looks like with Visual Studio code with the eslint extension installed and once merged (there's a default parameter now for this extension in the project), when you save:
![2018-12-10 13 54 18](https://user-images.githubusercontent.com/123822/49734033-57dfcc00-fc83-11e8-9b0d-f270b0fa1701.gif)

So you don't have to think about code formatting ever, it just does it for you.

Reply:
Well, the main reason that transformation folder exists is to contain demonstrations of the type of transformation files that could be passed to the `import` or `transfromlines` commands for example, that's the main reason they're .gitignored...also the reason I didn't want to eslint those files, as I want the code examples therein to be naive and accessible to anybody. Does that make sense?